src/components/Configure/content/fields/FieldMappings/ValuesFieldMapping.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useMemo, useState } from 'react';
+
+import { ComboBox } from 'src/components/ui-base/ComboBox/ComboBox';
+
+import { useSelectedConfigureState } from '../../useSelectedConfigureState';
+
+interface ValuesFieldMappingProps {
+  value: any,
```

Comment:
can you name this `providerValue` so it's more obvious what this is referring to?